### DISCUSSION TITLE
[New Feature] Remove nullable values from object

### DISCUSSION THREAD

**Discussion by bombillazo** [remeda/remeda#836] [Suggest a Function]
It would be cool to have a util that takes an object and removes all undefined/null values filtered out.

I've implemented this myself with this:

```ts
const filteredPayload = Object.fromEntries(
  Object.entries(payload).filter(
    ([, value]) => value != null && value !== undefined,
  ),
);
```

But it would be nice to have this type safe.

**Comment by eranhirsch** [comment]
This is available via ʼpickBy(isDefined)ʼ

**Comment by bombillazo** [comment]
Awesome to know, thanks!
